kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -5274,6 +5319,12 @@ def is_external_shared_storage():
 
         if cmd.useNuma or storage_migration_required:
             flag |= libvirt.VIR_MIGRATE_PERSIST_DEST
+        try:
+            tpm_devices = self.domain_xmlobject.devices.get_child_node_as_list('tpm')
+            if tpm_devices and len(tpm_devices) > 0:
+                flag |= libvirt.VIR_MIGRATE_PERSIST_DEST
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)